sqlup-mode.el

```diff
@@ -144,9 +144,8 @@ Other than <RET>, characters are in variable sqlup-trigger-characters."
 (defun sqlup-match-eval-keyword-p (dialect)
   "Return t if the code just before point ends with an eval keyword valid in
 the given DIALECT of SQL."
-  (cl-some 'identity
-           (mapcar #'(lambda (kw) (looking-back (concat kw "[\s\n\r\t]*") 0))
-                   (cdr (assoc dialect sqlup-eval-keywords)))))
+  (cl-some (lambda (kw) (looking-back (concat kw "\\s-*") 0))
```

Comment:
The `cl` package was added by a recent refactor, adding `some` for the first time. Up until then, I had written this entire package using only emacs-lisp. Maybe I should get familiar with the tools that `cl` provides me with, now that the cat's out of the bag.